client/js/components/day/employee.js

```diff
@@ -10,8 +11,8 @@ var Employee = React.createClass({
     return {
       name: '',
       rating: '',
-      gigID: Infinity,
-      employeeID: Infinity
+      gigId: Infinity,
```

Comment:
I used it as a default value for the properties expecting number values - can you elaborate more on what you mean by rationale/need? Are you asking why I set a default in the first place, or just why I used Infinity as opposed to other numbers?